applications/tasks/src/kz_tasks_scheduler.erl

```diff
@@ -233,9 +243,22 @@ handle_call({'start_task', TaskId}, _From, State) ->
             end;
         [#{started := Started}]
           when Started /= 'undefined' ->
+            lager:info("task ~s exists already", [TaskId]),
+            ?REPLY(State, {'error', 'already_started'})
+    end;
+handle_call({'restart_task', TaskId}, _From, State) ->
```

Comment:
@fenollp I found it useful from the command line to restart failing tasks while i was testing importing various CSVs.